scipy/sparse/csgraph/_flow.pyx

```diff
@@ -496,15 +496,16 @@ cdef bint _build_level_graph(
 
 cdef ITYPE_t _augment_paths(
         const ITYPE_t[:] edge_ptr,  # IN
-        const ITYPE_t current,  # IN
+        const ITYPE_t source,  # IN
         const ITYPE_t sink,  # IN
         const ITYPE_t[:] levels,  # IN
         const ITYPE_t[:] heads,  # IN
         const ITYPE_t[:] rev_edge_ptr,  # IN
         ITYPE_t[:] capacities,  # IN/OUT
         ITYPE_t[:] progress,  # IN
         ITYPE_t[:] flows,  # OUT
-        ITYPE_t flow  # OUT
+        ITYPE_t flow,  # OUT
+        ITYPE_t[:, :] stack
         ) nogil:
     """Computes blocking flow in layered graph using depth first search.
```

Comment:
Just noticed that this one also talks about blocking flow, when really, the outer loop is the one that checks for blocking-ness. Perhaps "Builds up blocking flow", or something like that, would be more descriptive.